server/api/recipes.js

```diff
@@ -1,7 +1,8 @@
+import { mapToPeapod } from '../../peapod/mapToPeapod';
```

Comment:
any reason for the import vs require thing here? Should we keep it consistent?

Reply:
Yeah, so there were some weird stubbing problems and that was the way I fixed it.

But I talked it through with Ben and figured out what I was doing wrong. So there's another pull request that makes all the require/imports consistent